packages/wrangler/src/dev.tsx

```diff
@@ -1003,3 +1042,26 @@ export function getBindings(
 
 	return bindings;
 }
+
+/**
+ * Merge two arrays of objects, overriding elements in `source` with
+ * elements from `override`, if they have the same `key` value
+ */
+function mergeWithOverride<T>(
```

Comment:
we should place this fn in a `utils`-like folder/file, but can't tell what would be appropriate, or what the convention is for these things. Suggestions welcome. Will also add tests once we settle on where it should live ^.^

Reply:
We have a `utils` folder in Wrangler where this could live?